js/core/templates/webidl-contiguous/method.html

```diff
@@ -1,3 +1,3 @@
 <span class='idlMethod' id="{{obj.idlId}}" data-idl data-title='{{name}}'>{{extAttr obj
-}}{{{special}}}{{#if obj.body}}<span class='idlMethType'>{{idlType obj.body}}</span> {{#if name
-}}<span class='idlMethName'>{{#tryLink obj}}{{name}}{{/tryLink}}</span>{{/if}}({{{children}}}){{/if}};</span>
\ No newline at end of file
+}}{{{special}}}{{#if obj.body}}<span class='idlType'>{{idlType obj.body}}</span> {{#if name
+}}<span class='idlName'>{{#tryLink obj}}{{name}}{{/tryLink}}</span>{{/if}}({{{children}}}){{/if}};</span>
```

Comment:
Careful with the line endings.

Reply:
(might be ok... but they've given us sadness before)

Reply:
I had to turn off insertFinalNewLine option on VSCode to keep the line endings intact 😅

I hope we get #1702 fixed soon to get rid of them!